snapshots/output/basic-project/lib/main.dart

```diff
@@ -95,7 +95,7 @@
 //                  ^^^ reference scip-dart pub dart:core 2.19.0 dart:core/`int.dart`/int#
 //                      ^^^^^ definition local 5
     Foo(1, value: true, value2: '');
-//  ^^^ reference scip-dart pub dart_test 1.0.0 lib/`other.dart`/Foo#
+//  ^^^ reference scip-dart pub dart_test 1.0.0 lib/`other.dart`/Foo#`<constructor>`().
```

Comment:
Now we're correctly referencing `<constructor>`, fixed by: https://github.com/Workiva/scip-dart/pull/165/files#diff-0ab533d80a9e11da530798d9b92364a97087c4754ab8251de00d85b635c6a4e4R76